init-tools.sh

```diff
@@ -157,7 +157,7 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     Tools-Override/crossgen.sh $__scriptpath/Tools
 
     echo "Making all .sh files executable under Tools."
-    ls $__scriptpath/Tools/*.sh | xargs chmod +x
+    find $__scriptpath/Tools/ -name "*.sh" -exec chmod +x {} \;
```

Comment:
For future reference, it is available on Fedora in the "findutils" package.

`dnf install findutils`